src/components/Calculator.js

```diff
@@ -1,6 +1,25 @@
 import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import calculate from './logic/calculate';
 
+function Result({ total, operation, next }) {
```

Comment:
- [ ] Good job implementing the use of props as the previous reviewer mentioned. Kindly use the ES6 syntax to create functions instead of using the ```function``` keyword. You can find more information on it [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)

Reply:
Hi @Ghiftee 

I tried changing the functions to arrow functions, but that does not seem possible because of the props, if I'm understanding this correctly.

I tried every example on the link you provided, but none seems to be working.